libs/@blockprotocol/type-system/crate/src/ontology/mod.rs

```diff
@@ -37,7 +37,8 @@ pub use shared::{
 // Re-export the repr contents so they're nicely grouped and so that they're easier to import in
 // a non-ambiguous way where they don't get confused with their non repr counterparts.
 // For example, `import crate::raw` lets you then use `raw::DataType`
-pub mod raw {
+#[allow(unreachable_pub, unused_imports)]
```

Comment:
why allow instead of fixing the lints?

Reply:
Because depending on the target (host/wasm) or build type (build/test) a different set is required.